src/ports/trendings/component.ts

```diff
@@ -1,53 +1,83 @@
 import seedrandom from 'seedrandom'
-import { Item, Sale, SaleFilters, SaleSortBy } from '@dcl/schemas'
-
-import { IMergerComponent } from '../merger/types'
+import { ISubgraphComponent } from '@well-known-components/thegraph-component'
+import { Item, Sale } from '@dcl/schemas'
 import { getDateXDaysAgo } from '../analyticsDayData/utils'
 import { IItemsComponent } from '../items/types'
 import { ITrendingsComponent, TrendingFilters } from './types'
+import {
+  fromTrendingSaleFragment,
+  getTrendingsQuery,
+  TrendingSaleFragment,
+} from './utils'
 
 const DEFAULT_SIZE = 20
+const MAX_RETRIES = 5
 export const SALES_CUT = 0.6
 export const VOLUME_CUT = 0.4
 
 export function createTrendingsComponent(
-  salesComponent: IMergerComponent<Sale, SaleFilters, SaleSortBy>,
+  collectionsSubgraph: ISubgraphComponent,
   itemsComponent: IItemsComponent
 ): ITrendingsComponent {
+  async function fetchTrendingSales(skip: number) {
+    const query = getTrendingsQuery(
+      { from: getDateXDaysAgo(2).getTime(), first: 1000, skip },
```

Comment:
This is more of a question than a change request, but why did we change to 2 days instead of 1? Doesn't the UI say `last 24hs`?